analyzer/src/funTest/kotlin/integration/DrupalIntegrationFunTest.kt

```diff
@@ -43,10 +43,9 @@ class DrupalIntegrationFunTest : AbstractIntegrationSpec() {
         binaryArtifact = RemoteArtifact.EMPTY,
         sourceArtifact = RemoteArtifact.EMPTY,
         vcs = VcsInfo(
-            VcsType.GIT,
-            "https://github.com/drupal/drupal.git",
-            "4a765491d80d1bcb11e542ffafccf10aef05b853",
-            ""
+            type = VcsType.GIT,
+            url = "https://github.com/drupal/drupal.git",
```

Comment:
Commit message:

> The intentation that `resolvedRevision` is only set by the downloader

"The intention to set `resolvedRevision` only by the downloader"

> also the the

"also the"

> , later commits can then address the issue of
> providing more detailed information about the revision in `VcsInfo`.

I propose to make this a separate sentence. Or maybe drop it completely, as we're not sure yet AFAIK which way we should go.